Microsoft.Toolkit.Uwp.UI/Extensions/HyperlinkExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Windows.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Documents;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Provides attached dependency properties for the <see cref="Hyperlink"/> content element that allows
+    /// it to invoke a <see cref="ICommand"/> when clicked
+    /// </summary>
+    public static class HyperlinkExtensions
+    {
+        private static void OnCommandPropertyChanged(DependencyObject sender, DependencyPropertyChangedEventArgs args)
+        {
+            Hyperlink hyperlink = sender as Hyperlink;
+
+            if (hyperlink != null)
+            {
+                ICommand command = args.NewValue as ICommand;
+
+                if (command != null)
+                {
+                    hyperlink.Click += OnHyperlinkClicked;
+                }
+                else
+                {
+                    hyperlink.Click -= OnHyperlinkClicked;
+                }
+            }
+        }
+
+        private static void OnHyperlinkClicked(Hyperlink sender, HyperlinkClickEventArgs args)
+        {
+            ICommand command = GetCommand(sender);
+            object parameter = GetCommandParameter(sender);
+
+            if (command != null)
+            {
+                command.Execute(parameter);
+            }
+        }
+
+        /// <summary>
+        /// Attached <see cref="DependencyProperty"/> for binding an <see cref="ICommand"/> instance to a <see cref="Hyperlink"/>
+        /// </summary>
+        public static readonly DependencyProperty CommandProperty = DependencyProperty.RegisterAttached("Command", typeof(ICommand), typeof(HyperlinkExtensions), new PropertyMetadata(null, OnCommandPropertyChanged));
```

Comment:
Fair enough, I kind of assumed it would be the same as a normal dependency property registration. Leave as is then.